README.md

```diff
@@ -157,13 +157,16 @@ The upper part of the popup contains the title, which can also be edited and sen
 
 ### Reviewing Diffs
 
-The `review` action will open a diff of the changes. You can leave comments using the `create_comment` action.
+The `review` action will open a diff of the changes. You can leave comments using the `create_comment` action or for multiline comments use `create_multiline_comment` in visual mode.
```

Comment:
Sure, for suggestion I used `<leader>glC`. In my personal config I remapped `summary` to `<leader>glS` and suggestion is `<leader>gls` but I'm not sure if we want to change these defaults because in both cases it makes perfect sense.